jupyterhub/app.py

```diff
@@ -365,12 +366,33 @@ def _log_format_default(self):
         """override default log format to include time"""
         return "%(color)s[%(levelname)1.1s %(asctime)s.%(msecs).03d %(name)s %(module)s:%(lineno)d]%(end_color)s %(message)s"
 
+    extra_log_file = Unicode("", config=True)
```

Comment:
can you add a `help="...` string? There should be one for every config=True trait, since it's what populates the `--help` output.